cli/dcoscli/util.py

```diff
@@ -86,6 +86,44 @@ def confirm_text(prompt, confirmation_text):
     return False
 
 
+def choicelist(choices, descriptions=[], msg='Please choose: '):
+    """
+    Allow to pick an item from a choicelist. For example :
```

Comment:
What is a choicelist?

Reply:
I'v just never heard the word choicelist before. I'm fine using it if it's a common term, I've just never heard it before.

Reply:
Not sure what you mean by standard, command-line tools such as `dialog` or `whiptail` use the term `menu` : 
> As  its  name  suggests, a menu box is a dialog box that can be used to present a list of choices in the form of a menu for the user to choose.

They also have the terms `list of choices` in the associated description, I've went for `choicelist` as it's more explicit than `menu` and clearly defines what that helper do. Let me know if you prefer `menu`.

Reply:
What about `prompt_with_choices()` since we actually block this call with a prompt.

Reply:
lgtm, I'll update the function name